### PR TITLE
refactor: [M3-6956] - MUI v5 Migration - `SRC > Components > FormControlLabel`

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝
- Component refactor for `<FormControlLabel />`
 
## Major Changes 🔄
- Use named export ✏️
- Add Storybook story 📄
- Move out of `core` folder 🗂️

## Preview 📷
Screenshot 2023-08-01 at 12 39 14 PM

## How to test 🧪
- Verify automated checks pass ✅
- Check that form labels still look and work as expected
  - For example, go check the `Set Number of Nodes` section on the Database Create page

### CODE REVIEW

**Review by jaalah-akamai — APPROVED** [review]

File: packages/manager/src/components/FormControlLabel.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    },
+  },
+  args: {
+    control: controlOptions.Checkbox,
+    label: 'This is a FormControlLabel',
+  },
+  render: (args) => <FormControlLabel {...args} />,
+};
+
+export default meta;
```
Comment (coliu-akamai) [inline packages/manager/src/components/FormControlLabel.stories.tsx:35]: just had a quick question -- is it convention to default export meta? Saw this on other stories.tsx files and wanted to ask about it

**Review by coliu-akamai — APPROVED** [review]
looks good to me! :D

File: packages/manager/src/components/FormControlLabel.stories.tsx (hunk 1 above)
Comment (bnussman-akamai) [reply packages/manager/src/components/FormControlLabel.stories.tsx:35]: Yes! Storybook CSF 3.0 (Storybook's Component Story Format) looks for a default export to get metadata about the component. We just follow their pattern for these Storybook stories.

https://storybook.js.org/docs/react/api/csf#default-export

Comment (coliu-akamai) [reply packages/manager/src/components/FormControlLabel.stories.tsx:35]: awesome, thanks for the info!

**Pull request merged**